packages/api-client/src/demo/demo.js

```diff
@@ -26,7 +26,7 @@ import {APIClient} from '@wireapp/api-client';
 import {AccessTokenStore} from '@wireapp/api-client/dist/commonjs/auth/';
 import {IndexedDBEngine} from '@wireapp/store-engine/dist/commonjs/engine';
 import ReactDOM from 'react-dom';
-import {WebSocketClient} from '@wireapp/api-client/dist/commonjs/tcp/';
+import {TOPIC as WebSocketTopic} from '@wireapp/api-client/dist/commonjs/tcp/';
```

Comment:
We should rename `TOPIC` to `WebSocketTopic` within `WebSocketClient` to be consistent with event naming and files from which they come from.